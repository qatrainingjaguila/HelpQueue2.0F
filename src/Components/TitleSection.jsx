import {useState} from 'react';
import {button} from 'react-bootstrap';
import { Button, Form, FormControl } from 'react-bootstrap';
import SearchResults from './pages/searchResults';


const TitleSection = () => {
  const [query,setQuery] = useState(``);

  const handleSubmit = (event) => {
    const form = event.target;
    event.preventDefault();
    console.log(form)
  }

return( 
    <>
    <section id = "title" class="title-section">
    {/* <img
          src="https://i.imgur.com/hxGZzW8.jpg"
          style = {{width:"100%"}}
        alt="image"/>
        <Form className="searchbar" onSubmit={handleSubmit}>
        <FormControl 
        type="text" 
        placeholder="Search QA Community Service..." 
        className="mr-sm-2"
        onInput={e => setQuery(e.target.value)}/> */}
        {/* </Form> */}
         <div class="container-fluid">


        <div class="row">
        <div class="col-lg-12"> 
        {/* <div class="col text-center"> */}
          {/* <button type="button" onclick="location.href='html/Signup.html'0"
            class="btn btn-lg btn-light explore-button">Explore</button> */}
        <br></br>   
        <br></br>
        <br></br>
        <br></br>
        <br></br>
        <br></br>
        <br></br>
        <br></br>
        <br></br>
        <br></br>
        <Form className="searchbar" onSubmit={handleSubmit}>
        <FormControl 
        type="text" 
        placeholder="Search QA Community Service..." 
        className="mr-sm-2"
        onInput={e => setQuery(e.target.value)}/>
        </Form>
        </div>
        </div> 
        </div>
    </section>
    {query && <SearchResults stateQuery={query}/>}
    </>
 );
}
export default TitleSection;