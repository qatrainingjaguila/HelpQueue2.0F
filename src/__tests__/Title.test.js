import renderer from 'react-test-renderer';
import TitleSection from '../Components/TitleSection';


test("Matches title snapshot", () => {
    const title = renderer.create(<TitleSection/>).toJSON();
    expect(title).toMatchSnapshot();
})
