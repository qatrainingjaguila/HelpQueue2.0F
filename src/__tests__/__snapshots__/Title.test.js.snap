// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches title snapshot 1`] = `
Array [
  <section
    class="title-section"
    id="title"
  >
    <div
      class="container-fluid"
    >
      <div
        class="row"
      >
        <div
          class="col-lg-12"
        >
          <br />
          <br />
          <br />
          <br />
          <br />
          <br />
          <br />
          <br />
          <br />
          <br />
          <form
            className="searchbar"
            onSubmit={[Function]}
          >
            <input
              className="mr-sm-2 form-control"
              onInput={[Function]}
              placeholder="Search QA Community Service..."
              type="text"
            />
          </form>
        </div>
      </div>
    </div>
  </section>,
  "",
]
`;
